pkg/logs/internal/tailers/file/tailer.go

```diff
@@ -157,15 +169,38 @@ func NewTailer(outputChan chan *message.Message, file *File, sleepDuration time.
 		stopForward:            stopForward,
 		isFinished:             atomic.NewBool(false),
 		didFileRotate:          atomic.NewBool(false),
-		info:                   info,
+		info:                   opts.Info,
 		bytesRead:              bytesRead,
 	}
+
+	if fileRotated {
+		addToTailerInfo("File Rotated", strconv.FormatBool(fileRotated), t.info)
```

Comment:
I think `File Rotated` is redundant if we also have `Last Rotation Date`. As `Last Rotation Date` existing implies that the file has already rotated. 

Nice addition BTW. having the date of the last rotation (rather than just a bool) will be useful.